torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -239,9 +239,9 @@
 {%- set tile_W = kernel.view(tile_W_3d, ["k_end - k_start", micro_gemm.register_blocking.block_n]) %}
 {%- else %}
     {%- if is_woq_int4 %}
-        {%- set tile_W = kernel.slice_nd(W, [("n_start", "n_start + n_size"), ("k_start * Nr / 2", "k_end * Nr / 2")]) %}
+        {%- set tile_W = kernel.slice_nd(W, [("nci * Nr", "(nci + 1) * Nr"), ("k_start * Nr / 2", "k_end * Nr / 2")]) %}
```

Comment:
Are there likely to be similar issues with other types ?

Reply:
Thanks for the comment. Feels like only `woq_int4` has this issue.